streams/src/main/java/org/apache/kafka/streams/internals/ApiUtils.java

```diff
@@ -18,43 +18,58 @@
 
 import java.time.Duration;
 import java.time.Instant;
-import java.util.Objects;
+
+import static java.lang.String.format;
 
 public final class ApiUtils {
+
+    public static final String MILLISECOND_VALIDATION_FAIL_MSG_FRMT = "The \"%s\" for \"%s\" is incorrect, value: %s.";
+
     private ApiUtils() {
     }
 
     /**
      * Validates that milliseconds from {@code duration} can be retrieved.
      * @param duration Duration to check.
-     * @param name Name of params for an error message.
+     * @param messagePrefix Prefix text for an error message.
      * @return Milliseconds from {@code duration}.
      */
-    public static long validateMillisecondDuration(final Duration duration, final String name) {
+    public static long validateMillisecondDuration(final Duration duration, final String messagePrefix) {
         try {
             if (duration == null)
-                throw new IllegalArgumentException("[" + Objects.toString(name) + "] shouldn't be null.");
+                throw new IllegalArgumentException(messagePrefix + " It shouldn't be null.");
```

Comment:
Nit: -> `messagePrefix + "It shouldn't be null.")` (remove space -- this allows to hand is an empty prefix without getting an undesired whitespace (ie, `""`, if prefix is not desired). Similar below.